index.js

```diff
@@ -844,9 +844,12 @@ function hexWrite (buf, string, offset, length) {
   }
   let i
   for (i = 0; i < length; ++i) {
-    const parsed = parseInt(string.substr(i * 2, 2), 16)
-    if (numberIsNaN(parsed)) return i
-    buf[offset + i] = parsed
+    const a = hexCharValueTable[string[i * 2]]
```

Comment:
why not bitshift here?

Reply:
Because it is in line with the style of the code it replaces.

Also, there is no performance difference.